src/preferences/preferences.hpp

```diff
@@ -180,6 +181,15 @@ struct option
 };
 };
 
+using std::string;
+#define SIMPLE_GETTER_SETTER(pref, type, def)             \
```

Comment:
Also since its used in a header file, the macro name should be more unique since its also defined in every file that includes that header.

Reply:
I could also undef it after the final use of the macro, I assume?